andela/workdey
# Ft view taskee profile from taskee list #128575691

# [#128575691] View Taskee Profile from taskee list
#### What does this PR do?

This PR implements the ability of a tasker to view a taskee profile from the taskee list page
#### How should this be manually tested?
- Log in as a tasker
- Click on `Taskee skillsets` button
- On the taskees view page, You should be able to click on a taskee's name
- You should see the taskee's name
#### Any background context you want to provide?

This is like an upgrade to a previous story. The story was the ability to view taskees and their skillsets. There was no functionality to view taskee's profile from that view
#### What are the relevant pivotal tracker stories?
#128575691
